contracts/oss-diff.yml

```diff
@@ -5793,6 +5750,162 @@ components:
           type: integer
           format: int64
           minimum: 0
+    Tasks:
+      type: object
+      properties:
+        links:
+          readOnly: true
+          $ref: '#/paths/~1dashboards/get/responses/200/content/application~1json/schema/properties/links'
```

Comment:
I am a little unsure about this bit here @glinton. If there are other things I would need to do to fix this up.

Reply:
Ah yes, the `-diff` files are nice to have, but are in no way a prioirty and haven't gotten any love. they aren't used by the ui and their terrible generator `oats` as yet, so we ignore those inline references. The solution would be to go through and add an entry for the schemas, responses, etc that is being used. Could be done simply/hacky in a similar pattern to the `_schemas.yml` `bash` almost template thing we do now, but isn't necessary.